### DISCUSSION TITLE
Badge label variant

### DISCUSSION THREAD

**Discussion by holiq** [saadeghi/daisyui#3119] [Ideas / feature requests]
Hello, I have custom badge variant and i called it badge-label, I think it would be great if it became official Daisyui, so can I make a PR for this new variant?

![image](https://github.com/saadeghi/daisyui/assets/30729921/16928a00-187e-4143-bf4b-2909faa3cceb)

**Comment by saadeghi** [comment]
This might look good visually but the background/foreground color contrast is not accessible, and hard to read for many people.

**Reply by holiq** [reply]
if I think its right too, so what should I do?

I create the variant like this
![image](https://github.com/saadeghi/daisyui/assets/30729921/1225e908-46b9-4f23-9eb2-0f30b58b916a)
